src/leo_object_storage_server.erl

```diff
@@ -301,6 +302,11 @@ get_eof_offset(Id) ->
     gen_server:call(Id, get_eof_offset, ?DEF_TIMEOUT).
 
 
+%% @doc Update each diskspace's status
+update_diskspace_status(Id, DiskSpaceStatus) ->
+    gen_server:call(Id, {update_diskspace_status, DiskSpaceStatus}, ?DEF_TIMEOUT).
```

Comment:
Catch the exception that can be raised when get_server:call get timeout to prevent the caller  leo_object_storage_diskspace_mon from being crashed OR
Change the ?DEF_TIMEOUT to infinity to make the timeout never happen.